.gitignore

```diff
@@ -24,6 +24,8 @@ integration-tests/logs
 integration-tests/smoke/logs
 integration-tests/soak/logs
 tmp-manifest-*
-artifacts
+/artifacts
```

Comment:
only cache the root directory artifacts folder (we commit to source other artifacts/ folders)